sinks/splunk/splunk.go

```diff
@@ -435,10 +435,15 @@ func (sss *splunkSpanSink) Ingest(ssfSpan *ssf.SSFSpan) error {
 		return err
 	}
 
-	// choose (1/spanSampleRate) spans for sampling if any spans
-	// have the traceID of 0 or are declared indicator spans, they
-	// will always be chosen, regardless of the sample rate.
-	if !ssfSpan.Indicator && ssfSpan.TraceId%sss.spanSampleRate != 0 {
+	// wouldDrop indicates whether this span would be dropped
+	// according to the sample rate. (1/spanSampleRate) spans
+	// will be kept. spans with a traceID of 0 will always be
```

Comment:
The comment about TraceID 0 is technically true, but we discard those spans via the [`ValidateTrace`](https://github.com/stripe/veneur/blob/5edc4c97de5e13cfb41ff2c4b669a3bb241c225d/protocol/wire.go#L80-L98) call above; the comment should be removed. (:

Reply:
Ah, I kept it from before and also saw spawns with ParentID of 0 so I didn't connect that this was a different ID